sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -547,8 +547,12 @@ def _get_required_container_version(job_type=None):
     version = pkg_resources.get_distribution(GOOGLE_PACKAGE_NAME).version
     # We drop any pre/post parts of the version and we keep only the X.Y.Z
     # format.  For instance the 0.3.0rc2 SDK version translates into 0.3.0.
-    container_version = (
-        '%s.%s.%s' % pkg_resources.parse_version(version)._version.release)
+    parsed_version = pkg_resources.parse_version(version)
+    try:
+      container_version = (
+        '%s.%s.%s' % parsed_version._version.release)
+    except:
```

Comment:
What is the exception here? Something like NameError?

Reply:
`AttributeError`